taskana-adapter-camunda-spring-boot-sample/src/main/resources/application.properties

```diff
@@ -39,7 +39,7 @@ taskana-system-connector-camundaSystemURLs=http://localhost:8081/example-context
 ####################################################################################
 #
 # Configure the datasource for Taskana DB (used by taskana-connector)
-taskana.datasource.jdbcUrl = jdbc:h2:tcp://localhost:9090/mem:taskana;IGNORECASE=TRUE;LOCK_MODE=0;
```

Comment:
Why did the port change?

Reply:
The port is correlated to the port in the TASKANA Spring Boot REST example (see ExampleRestConfoguration there), if using h2 as an in memory database for quick plug & play. We decided on port 9095 because 9090 is too regular and is more likely to be already in use.

Reply:
alright.